armi/reactor/reactors.py

```diff
@@ -2009,8 +2013,7 @@ def updateAxialMesh(self):
                         for h in [(b.p.ztop - b.p.zbottom) / b.p.axMesh] * b.p.axMesh
                     ]
                     for a in self
-                    if self.findAllAxialMeshPoints([a])
-                    == self.findAllAxialMeshPoints([refAssem])
+                    if self.findAllAxialMeshPoints([a]) == refMesh
```

Comment:
Also pretty sketchy to be comparing a list of floats like this. Nothing new though